website/src/components/canvas/CanvasList.jsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * @file HTML generation for the canvas list
```

Comment:
This entire file should live in the components/home folder rather than components/canvas, since it's involved in the home page display.
Right now the folders are structured based on page, so everything in canvas is specifically related to the actual canvas grids.